pkg/apis/naiserator/v1alpha1/application.yaml

```diff
@@ -18,10 +18,10 @@ spec:
       properties:
         spec:
           required:
-            - image
-            - team
-            - port
-            - healthcheck
+          - image
+          - team
+          - port
+          - liveness
```

Comment:
Burde vel kanskje ha `readiness` her også.

Reply:
as discussed, we'll make both liveness and readiness optional